assets/js/index.js

```diff
@@ -0,0 +1,92 @@
+// TODO: Include packages needed for this application
+const inquirer = require("inquirer");
```

Comment:
good job listing all fo your external packages/methods at the top of the file! This is great for readability